angr/analyses/decompiler/decompiler.py

```diff
@@ -243,6 +243,7 @@ def progress_callback(p, **kwargs):
             self.func,
             seq_node,
             cfg=self._cfg,
+            clinic=clinic,
```

Comment:
Let's pass in the output that Clinic generates, instead of passing in the entire Clinic instance.